src/constants/blacklistConfig.js

```diff
@@ -140,6 +145,38 @@ const programs = {
 		'next_stage',
 		'created_at',
 		'updated_at',
+		'is_under_edit',
+		'deleted_at',
+		'metaData',
+		'meta',
+		'stage',
+		'organizations',
+		'is_comments',
```

Comment:
@priyanka-TL what is this key for?

Reply:
is_comments is the key that we are sending to identify the resources in the program have comments or not, that need not to be stored in the blob, It will add based on the comments.

Reply:
Can we rename it has_open_comments?

Reply:
Hence listing also we are using the same key, we need to inform the frontend team also, that can we taken up in the next release?